data/json/items/resources/chemicals.json

```diff
@@ -33,6 +33,57 @@
     "count": 250,
     "ammo_type": "components"
   },
+  {
+    "type": "AMMO",
+    "id": "chem_calcium_chloride",
+    "category": "chems",
+    "price": 3000,
+    "price_postapoc": 500,
+    "name": { "str_sp": "calcium chloride" },
+    "symbol": "=",
+    "color": "white",
+    "description": "A white odorless strongly hygroscopic powder with an extremely salty taste.  Often used as a de-icing agent or as dessicant.",
+    "container": "bag_canvas",
+    "material": [ "powder_nonflam" ],
+    "volume": "250 ml",
+    "weight": "2150 mg",
+    "ammo_type": "components",
+    "count": 250
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_washing_soda",
+    "category": "chems",
+    "price": 3000,
+    "price_postapoc": 500,
+    "name": { "str_sp": "washing soda" },
+    "symbol": "=",
+    "color": "white",
+    "description": "A white odorless hygroscopic powder.  Often used as a cleansing agent and for glass manufacturing, among other things.  Also known as sodium carbonate or soda ash.",
+    "container": "box_small",
+    "material": [ "powder_nonflam" ],
+    "volume": "250 ml",
+    "weight": "2540 mg",
+    "ammo_type": "components",
+    "count": 250
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_baking_soda",
```

Comment:
Couldn't this be an alternative to yeast in many food recipes? Might be worthwhile to add later.

Reply:
It's not a simple substitution like that, you'd also need to add some kind of acid to use baking soda for this.